rdr_service/dao/bq_participant_summary_dao.py

```diff
@@ -773,6 +775,31 @@ def datetime_to_date(val):
         data['distinct_visits'] = len(dates)
         return data
 
+    def _calculate_test_participant(self, summary):
+        """
+        Calculate if this participant is a test participant or not.
+        :param summary: summary data
+        :return: dict
+        """
+        test_participant = summary['is_ghost_id']
+
+        # Check for @example.com in email address
+        if not test_participant:
+            # Check to see if the participant is in the Test HPO.
+            if (summary.get('hpo') or 'None').lower() == 'test':
+                test_participant = 1
+            # Test if @example.com is in email address.
+            elif '@example.com' in (summary.get('email') or ''):
+                test_participant = 1
+            # Check for SMS phone number for test participants.
+            elif '4442' in re.sub('[\(|\)|\-|\s]', '', (summary.get('login_phone_number') or 'None')):
+                test_participant = 1
+            elif '4442' in re.sub('[\(|\)|\-|\s]', '', (summary.get('phone_number') or 'None')):
+                test_participant = 1
```

Comment:
This could flag participants with phone numbers like  xxx-x44-42xx

Reply:
You are correct, good catch.  I'll fix that up.